explorer/app/components/common/Button/button.tsx

```diff
@@ -5,28 +5,32 @@
 
 // Core dependencies
 import { Button as MButton } from "@mui/material";
-import React from "react";
+import React, { forwardRef } from "react";
 
 // App dependencies
 import { Button as ButtonProps } from "./common/entities";
 
 type Props = ButtonProps;
 
-export const Button = ({
-  children,
-  disabled = false,
-  EndIcon,
-  StartIcon,
-  ...props /* Spread props to allow for Button specific props ButtonProps e.g. "onClick". */
-}: Props): JSX.Element => {
+export const Button = forwardRef<HTMLButtonElement, Props>(function Button(
```

Comment:
Can you give me a bit of background on why the `forwardRef` is required here?

Reply:
The next Link is passing a ref to the button, and as the Button component is a custom component comprising of `MuiButton` we need to forward the ref on.